src/hub.js

```diff
@@ -1635,3 +1638,14 @@ document.addEventListener("DOMContentLoaded", async () => {
   authChannel.setSocket(socket);
   linkChannel.setSocket(socket);
 });
+
+/*
+Like ES6 dynamic import() but for multiple modules.
+Modules are downloaded in parallel and executed in sequence
+*/
+function importAll(urls) {
+  const moduleStr = urls.map(url => `import '${url}';`).join("\n");
```

Comment:
let's just return a resolved promise (`Promise.resolve()`) if the urls array is empty